packages/inter-protocol/src/proposals/utils.js

```diff
@@ -108,6 +110,7 @@ export const makeInstallCache = async (
   };
 
   const wrapInstall = install => async (mPath, bPath, opts) => {
+    // @ts-expect-error xxx bundle types
```

Comment:
What is the issue here?

Reply:
> Property 'endoZipBase64Sha512' does not exist on type 'EndoZipBase64Bundle | GetExportBundle | NestedEvaluateBundle'.

Reply:
That seems like a legitimate issue. Why suppress it?
Or is it because of an overload in `loadBundle` which isn't typed correctly?

Reply:
The type was wrong (which is what I meant by "xxx bundle types")

e6c5a934a5ab6af203151336c2eeb22aa096faf1 has a fix

Reply:
I believe runtime changes are worth when they directly address issues found by typing. improvements